src/main/scala/com/greencatsoft/angularjs/internal/Angular.scala

```diff
@@ -12,6 +14,8 @@ import com.greencatsoft.angularjs.Filter
 @js.native
 private[angularjs] trait Angular extends js.Object {
 
+  def injector(): Injector = js.native
```

Comment:
Wouldn't it be better if we omit the parenthesis here to make it consistent with its counterpart in `angularjs.Angular.injector`?